ir_actions_server_security_adj/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module gives read access for server actions to the internal users.
+
+Background
+~~~~~~~~~~
+
+Server actions should be restricted exclusively to internal users and
```

Comment:
Research into past history reveals that it was adjusted to allow internal users to perform server actions.
It is used for that purpose, and  to allow non-internal users to run it as well. (using sudo().).

Reply:
@kanda999  I still don't get the purpose of this adjustment. I think all users have read access to model_ir_actions_server. You can see in here. https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/security/ir.model.access.csv#L86

Reply:
So, I think my current readme is correct.